setup.py

```diff
@@ -142,9 +140,30 @@ def check_for_openmp(cc_var):
 
 
 # == Getting tofu version =====================================================
+
 _HERE = os.path.abspath(os.path.dirname(__file__))
-def get_version_tofu(path=_HERE):
 
+
+def updateversion(path=_HERE):
```

Comment:
Yes and no...
I agree having updateversion defined in a standalone side module is a bit cumbersome because it seems it is only needed in the setup.py.

But this was done in order to allow travis (or us) run updateversion as a standalone without having to import setup.py (which would run it).

updateversion can be useful as a standalone when we want to force an update of the version number prior to running the setup.py, using for example:

> python -c "import _updateversion as up; out = up.updateversion(); print(out)"

This is not possible anymore with this change.

Are you sure there is no other fix for issue #292 ?  It used to work before, I don't get why it stopped working

Reply:
Okay, thanks for reviewing. So it seems this should not be merged as such.
I believe then this needs more investigation. 
I have no idea why it's broken under Windows. Too bad...

Reply:
Nice,
Please also make sure the PR does not include the chages from PR #292 (i.e.: updateversion)
These are two independent changes, and as we discussed, I'm sure sure we can merge PR #292 as of now.
I suspect it is because you created your branch from the branch you had already created for PR #292 , right ?

Best practice : always branch from devel, a world where things are stable and validated, to make small incremental changes ;-)